.github/workflows/action.yml

```diff
@@ -28,7 +28,9 @@ jobs:
           sudo apt update
           sudo apt-get install -y libgnutls28-dev libcurl4-openssl-dev libssl-dev
           wget https://raw.githubusercontent.com/SatelliteQE/broker/master/broker_settings.yaml.example
-          pip install -U --no-binary=pycurl -r requirements.txt -r requirements-optional.txt
+          pip uninstall -y pycurl
+          pip install --compile --no-cache-dir pycurl
```

Comment:
not blocking, but if for some reason you need to update this pr (other feedback) can you add a comment as to why these steps are necessary?

Reply:
ack, added.